pkg/v1/kubeoptions/schemas.go

```diff
@@ -3,8 +3,8 @@ package kubeoptions
 // View represents list of feature-gates/admission-controllers by kubernetes version.
 type View struct {
 	// KubeVersion represents the Kubernetes minor version in format: "X.Y".
-	KubeVersion string `json:"KubeVersionMinor"`
+	KubeVersion string `json:"KubeVersionMinor"` //nolint:tagliatelle
```

Comment:
I think it will be better to remove `tagliatelle` linter from the config with its settings.
Also we can do the same for `goerr113` linter, because it seems to be useless for this repo.